subsys/usb/class/netusb/netusb.c

```diff
@@ -21,27 +21,27 @@
 #include "../../usb_descriptor.h"
 #include "../../composite.h"
 #include "netusb.h"
-#include "eth_emu.h"
 #include "function_ecm.h"
 
 #define NETUSB_NUM_CONF 1
 
 static struct __netusb {
+	struct net_if *iface;
 	u8_t conf;
```

Comment:
I know the conf variable is not part of your patch so this could be "fixed" in some later patch. It just looks like the conf variable could be clarified it a bit. It looks like a boolean value so we could change the u8_t to bool. This would mean that the use of it would change from 0 -> false and 1 -> true.

Reply:
Original idea about conf is to keep USB selected configuration value (or offset of selected configuration in the conf array)